overlay/base.yml

```diff
@@ -24,5 +24,8 @@ params:
     system_domain:   (( grab params.system_domain ))
     deployment_name: (( grab name ))
     network_name:    (( grab params.network ))
+    cf_core_network:    (( grab params.cf_core_network || params.bosh-variables.network_name ))
```

Comment:
These should be part of the ops file, as they're only relevant if that ops file is used